core/Node.go

```diff
@@ -144,6 +144,8 @@ func (n *Node) Message() proto.Message {
 func (n *Node) GetValue(url string) (v reflect.Value, e error) {
 	root, sub := lib.URLShift(url)
 	switch root {
+	case "/type.googleapis.com":
+		fallthrough
```

Comment:
Slightly nitpicky, but these should probably be inverted so that it checks "type.googleapis.com" before "/type..." (and the same with Services.  This will reduce the number of string compares by one in the most probable case (i.e. no "/").

Reply:
Makes sense. Fixed with https://github.com/hpc/kraken/pull/162/commits/8f83313fed69ec9f9d1c091d544a3c730c1d7880